src/transaction.js

```diff
@@ -134,9 +134,18 @@ function getContractColdFields(cold_tx, network_byte, acc) {
             cold_tx['contractInitExplain'] = 'Create token' + (contract_type === 'TOKEN_CONTRACT' ? ' ' : ' (support split) ') + 'with max supply ' + BigNumber(init_data[0]['value']).dividedBy(init_data[1]['value']);
             cold_tx['contractInitTextual'] = "init(max=" + BigNumber(init_data[0]['value']).dividedBy(init_data[1]['value'])+ ",unity= "+ BigNumber(init_data[1]['value']) + ",tokenDescription='" + init_data[2]['value'] + "')";
             break;
-        case 'PAYMENT_CONTRACT': case 'LOCK_CONTRACT': case 'NON_FUNGIBLE_TOKEN_CONTRACT':
-            cold_tx['contractInitExplain'] = ''
-            cold_tx['contractInitTextual'] = ''
+        case 'PAYMENT_CONTRACT':
+            cold_tx['contractInitExplain'] = 'Register Payment Channel Contract';
```

Comment:
The explain is belong to application level setting, not in SDK level. Please move all explains in wallet project.